lib/pacemaker/pcmk_sched_ordering.c

```diff
@@ -863,13 +867,14 @@ order_rsc_sets(const char *id, xmlNode *set1, xmlNode *set2,
     const char *action_1 = crm_element_value(set1, "action");
     const char *action_2 = crm_element_value(set2, "action");
 
-    const char *sequential_1 = crm_element_value(set1, "sequential");
-    const char *sequential_2 = crm_element_value(set2, "sequential");
+    enum pe_ordering flags = pe_order_none;
 
-    const char *require_all_s = crm_element_value(set1, "require-all");
-    bool require_all = require_all_s? crm_is_true(require_all_s) : true;
+    bool require_all = false;
+    int rc = pcmk__xe_get_bool_attr(set1, "require-all", &require_all);
 
-    enum pe_ordering flags = pe_order_none;
+    if (rc == EINVAL || rc == ENODATA) {
+        require_all = true;
+    }
```

Comment:
I think just initializing require_all to true and ignoring rc would do the right thing here

Reply:
Yep, this works (and allows me to entirely get rid of rc here).